src/Benchotron/UI/Console.purs

```diff
@@ -94,9 +95,10 @@ runBenchmarkConsole benchmark = do
   lift $ do
     stderrWrite "\n"
     noteTime \t -> "Finished at: " <> t <> "\n"
-  return r
+  pure r
   where
-  noteTime f = now >>= toLocaleTimeString >>= (stderrWrite <<< f)
+  noteTime f = nowString >>= (stderrWrite <<< f)
+  nowString = JSD.toTimeString <$> JSD.fromDateTime <$> DDI.toDateTime <$> now
```

Comment:
Either way makes my eyes bleed :-) Get the milliseconds from native code, pull into the PS world, typify it, run the conversion, convert it back to native format and immediately shove back to the native world to convert to native string and return.